docs/source/queries/values.md

```diff
@@ -55,6 +56,15 @@ session.query("INSERT INTO ks.tab (a) VALUES(?)", (2_i32,)).await?;
 session
     .query("INSERT INTO ks.tab (a, b) VALUES(?, ?)", &(&2_i32, &"Some text"))
     .await?;
+
+// A map of named values can also be provided:
+let mut vals: HashMap<String, CqlValue> = HashMap::new();
```

Comment:
This example can easily be missed, what do you think of extracting it to another paragraph (in a similar fashion to `NULL values`)?

Reply:
It's done on purpose - I actually **don't** want to encourage people to use named bind markers, since unnamed markers are unequivocally better. I put the example here just for completeness.